### ISSUE TITLE
Remove unused `alt-hero` field from open-community-survey.md project file

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3208]
### Overview
As a developer, I want to make sure unused code is removed from the codebase in project cards so that
- time to load the website is as low as possible,
- it's easier to understand the code, and
- time to review the code is reduced.

### Details
In project md files, `image-hero` is a decorative background image and `alt-hero` is the alt text associated with `image-hero`. However, `alt-hero` text is not used anywhere in our codebase since the `image-hero` is used as an SCSS background image.

Moreover, if the `image-hero` were to be used in an img HTML tag in the future, the alt text for all the `image-hero` images would be `alt=""`, which could be set programmatically for all the alt properties associated with each `image-hero`.   

### Action Items
- [x] For the file `_projects/open-community-survey.md`, remove line 8 for the alt-hero
```
alt-hero: 'Silhouette of buildings and houses with different colors, shapes and styles.'
```
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #2722 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2722 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #2722 

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_projects/open-community-survey.md`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/c8a48d402a6472538dba4c279c88a3e947e05fd1/_projects/open-community-survey.md?plain=1#L8

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

**Comment by Skydodle** [comment]
Availability: Wednesdays, Fridays & Saturday

ETA: By Sunday 11/13 end of day

**Comment by ExperimentsInHonesty** [comment]
This issue is QAed and is fine.

While I was QAing this issue, I realized that there is also alt text in the markdown project files for the image we use on the homepage and on the mini card.  
1. We have to make an epic to clean those up so that they are WCAG compliant in case we do need to use them
2. Some of the images contain text that is not conveyed in the text below the images on the home & project pages (heart, vrms, etc.).  We are using the alt text on the home and project page (same include).

**Comment by ExperimentsInHonesty** [comment]
This whole thing was resolved by changing the alt text for vrms to "Volunteer Relationship Management (VRMS)" in the spreadsheet related to this issue.
https://github.com/hackforla/website/issues/2613

There are no other outstanding items.
